src/Blocks/continuous.jl

```diff
@@ -12,11 +12,15 @@ Initial value of integrator state ``x`` can be set with `x`
 # Parameters:
 
   - `k`: Gain of integrator
+
+# Unknowns:
+
+  - `x`: State of Integrator (with initial guess of 0.0)
 """
 @mtkmodel Integrator begin
     @extend u, y = siso = SISO()
     @variables begin
-        x(t) = 0.0, [description = "State of Integrator"]
+        x(t), [description = "State of Integrator", guess = 0.0]
```

Comment:
There is no guess needed here, this should be a default value.

Reply:
Resolved with: https://github.com/SciML/ModelingToolkitStandardLibrary.jl/pull/274/commits/3a7ccfbd43de07b1f1de3f81dc1b86e173146eac